testnets/neutrontestnet/chain.json

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "neutrontestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Neutron Testnet",
+  "chain_id": "baryon-1",
+  "bech32_prefix": "neutron",
```

Comment:
Please register this prefix to https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
It was registered and already merged https://github.com/satoshilabs/slips/pull/1493